Ignore new failing tests

jyellick opened the pull request:
All but one of these failing tests is for the fault prover which depends on raw Geth DB access, so the failures are anticipated.  The other is for the Dencun fork behavior, which I believe our Erigon is too old to
  handle.  Once Erigon is updated, we should be able to enable it.
  
  The one remaining failure is legitimate and requires an Erigon fix that will be addressed in a separate PR

wsdt commented:
Added a better error message

wsdt commented:
ToDo:

```
so main things are (1) clarify that people need Rinkeby ETH in the first place otherwise they can't do anything (2) when user rejects transaction, show the rror message and stop the spinner, and (3) deal with the reject -transaction bug , and (4) intercept the error message from the smart contract and replace it with "CAPTCHA incorrect" in the front end



basically, if they reject the metamask transaction it generates an error because message is zero or something like that



I think it keeps waiting for the server resonse
```

jyellick committed:
Ignore new failing tests

All but one of these failing tests is for the fault prover which depends
on raw Geth DB access, so the failures are anticipated.  The other is
for the Dencun fork behavior, which I believe our Erigon is too old to
  handle.  Once Erigon is updated, we should be able to enable it.

jyellick committed:
Use updated Erigon with genesis write fix
